lua/weapons/weapon_monke.lua

```diff
@@ -1,5 +1,4 @@
--- This file is subject to copyright - contact swampservers@gmail.com for more information.
```

Comment:
these lines need to be here or it will try to install it in fatkid

Reply:
oh whoops. fixed